tests/TestCase/Database/Schema/PostgresSchemaTest.php

```diff
@@ -990,9 +989,11 @@ public function testConstraintSql($name, $data, $expected)
         $table = (new TableSchema('schema_articles'))->addColumn('title', [
             'type' => 'string',
             'length' => 255,
-        ])->addColumn('author_id', [
-            'type' => 'integer',
-        ])->addConstraint($name, $data);
+            ])
+        ->addColumn('author_id', [
+                'type' => 'integer',
+                ])
+        ->addConstraint($name, $data);
```

Comment:
What's going on with the formatting here?

Reply:
Sorry phpcs asked me to make these changes...

Reply:
Removed whitespaces. I had phpcs installed with my IDE and I made changes exactly as it recommended. Reverted. Please find the next commit.